styles/_status-label.scss

```diff
@@ -78,7 +79,7 @@
   &-solved,
   &-new,
   &-hold,
-  &-answered {
+  &-pending {
     text-transform: lowercase; // To be consistent with Lotus
```

Comment:
Not related with this PR but I think this comment is obsolete 😄

Reply:
No, I don't think it is. Well at least it might be, but I don't see how that can be concluded from here.

Reply:
`Support` doesn't lowercase labels anymore. But this should be addressed separately as part of a different issue: https://zendesk.atlassian.net/browse/HCJ-2459

Reply:
👍